Components/Navigation.html

```diff
@@ -0,0 +1,243 @@
+                <li class="nav-item">
+                  <a class="nav-link" href="#">
+                    <ion-icon name="mail-outline" class="nav-icon"></ion-icon
+                  ></a>
+                </li>
+              </ul>
+            </nav>
+          </div>
+
+          <h3 class="showCodeToggle" onclick="showCodeToggle()"></h3>
+
+          <div class="code hideCode">
+            <iframe
+              src="https://carbon.now.sh/embed/e0TfiUZxbvij381bZcgS"
+              style="
+                width: 100%;
+                height: 625px;
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
+              "
+              sandbox="allow-scripts allow-same-origin"
+            >
+            </iframe>
+          </div>
+        </div>
+
+        <!-- Navigation Bar -->
+
+        <div class="exampleCont">
+          <h3 class="exampleText">Navigation Bar</h3>
+
+          <div class="flexCont">
+            <nav class="navbar-cont">
+              <ul class="navbar">
+                <li class="navbar-item">
+                  <a class="nav-link nav-logo" href="#">
+                    <ion-icon name="cloud-outline"></ion-icon>
+                    <span class="nav-text"> Logo</span>
+                  </a>
+                </li>
+                <div class="navbar-items-cont">
+                  <li class="navbar-item">
+                    <a class="nav-link" href="#">
+                      <ion-icon
+                        name="search-outline"
+                        class="navbar-icon"
+                      ></ion-icon
+                    ></a>
+                  </li>
+                  <li class="navbar-item">
+                    <a class="nav-link" href="#">
+                      <ion-icon
+                        name="notifications-outline"
+                        class="navbar-icon"
+                      ></ion-icon
+                    ></a>
+                  </li>
+
+                  <li class="navbar-item">
+                    <a class="nav-link" href="#"
+                      ><span class="nav-text"> Products</span></a
+                    >
+                  </li>
+                  <li class="navbar-item">
+                    <a class="nav-link" href="#"
+                      ><span class="nav-text"> Cart</span></a
+                    >
+                  </li>
+                </div>
+              </ul>
+            </nav>
+          </div>
+
+          <h3 class="showCodeToggle" onclick="showCodeToggle()"></h3>
+
+          <div class="code hideCode">
+            <iframe
+              src="https://carbon.now.sh/embed/NLThxFlhpxmURJkHoe54"
+              style="
+                width: 100%;
+                height: 834px;
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
+              "
+              sandbox="allow-scripts allow-same-origin"
+            >
+            </iframe>
```

Comment:
This iframe showing code snippit of Avatar component
![Screenshot (60)](https://user-images.githubusercontent.com/42497931/112653920-e38dbe80-8e74-11eb-9340-b967dd167cb8.png)

Reply:
ok, got it, These things are happening because carbon issued duplicate embed codes.
Going to Fix it.

Reply:
Fixed